testsuite/MDAnalysisTests/core/test_accumulate.py

```diff
@@ -0,0 +1,144 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+from __future__ import division, absolute_import
+
+import numpy as np
+from numpy.testing import assert_equal, assert_almost_equal
+
+import MDAnalysis as mda
+from MDAnalysis.exceptions import DuplicateWarning, NoDataError
+from MDAnalysisTests.datafiles import PSF, DCD, TPR_xvf, TRR_xvf, GRO
+import pytest
+
+levelist=('atoms', 'residues', 'segments')
+        
+class TestAccumulate(object):
+    """Tests the functionality of *Group.accumulate()."""
+    @pytest.fixture(params=levelist)
+    def group(self, request):
+        u = mda.Universe(PSF, DCD)
+        return getattr(u, request.param)
+
+    def test_accumulate_str_attribute(self, group):
+        assert_almost_equal(group.accumulate("masses"), np.sum(group.atoms.masses))
+
+    def test_accumulate_different_func(self, group):
+        assert_almost_equal(
+            group.accumulate("masses", function=np.prod),
+            np.prod(group.atoms.masses))
+
+    @pytest.mark.parametrize('universe, name, compound',
+                             ((mda.Universe(PSF, DCD), 'resids', 'residues'),
```

Comment:
this has the same problem, the scope of the Universe objects here will be the entire test run.  The advantage of fixtures is the object only lives as long as the test function

Reply:
I just replaced the DCD / TPR universes with the infamous `UnWrapUniverse` for the compound tests. Creating `UnWrapUniverse`s is so fast that fixtures won't improve anything here.
Hope you're ok with this.